test/qa/test_firewall6.py

```diff
@@ -111,7 +111,7 @@ def test_firewall_enable_connect(tech, proto, obfuscated):
 
 
 @pytest.mark.parametrize("tech,proto,obfuscated", lib.OVPN_STANDARD_TECHNOLOGIES)
-@pytest.mark.parametrize("port", lib.PORTS)
+@pytest.mark.parametrize("port", lib.PORTS, ids=[lib.Protocol.UDP, lib.Protocol.TCP, lib.Protocol.ALL])
```

Comment:
there is no relation between `ids` and `PORTS`. If we add more ports, this will show incorrect logs